scripts/releases/prebuild/cli.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ * @oncall react_native
+ */
+
+const {dependencies, platforms} = require('./configuration');
+const yargs = require('yargs');
+
+/*::
+import type {Dependency, Platform} from './types';
+import {skip} from "rxjs/operators";
+*/
+
+const arrayLike = (value /*: Array<any> */) /*: Array<any> */ =>
+  Array.isArray(value) ? value : [value];
+
+const cli = yargs
+  .usage(
+    'This script prepares iOS prebuilds for React Native. It downloads the dependencies, prepare them, builds them and creates the XCFrameworks.' +
+      'Calling the script with no options will build all the dependencies for all the slices and configurations.',
+  )
+  .option('setup', {
+    alias: 's',
+    type: 'boolean',
+    describe: 'Download and setup dependencies',
```

Comment:
what's the default? The question holds for all the items below

What I don't like is that to do everything, we need to call
```
node <script> -s -w -b -c
```

I would rather have the default case that does everything.

Reply:
Default case runs all tasks... But that is probl. not super clear? The initial doc says "Calling the script with no options will build all the dependencies for all the slices and configurations."